fiware-region-sanity-tests/tests/fiware_region_object_storage_tests.py

```diff
@@ -171,16 +173,16 @@ def test_create_big_object_and_download_it_from_container(self):
         big_file_url_1 = self.conf[PROPERTIES_CONFIG_TEST][PROPERTIES_CONFIG_SWIFT][PROPERTIES_CONFIG_SWIFT_BIG_FILE_1]
         f = urllib2.urlopen(big_file_url_1)
```

Comment:
You should not include the operations:
- Open file
- Write content to the file (append)

Between a try except block?